messages/4.16.0.txt

```diff
@@ -0,0 +1,9 @@
+SublimeLinter 4.16 - Quick Actions (beta)
+
+We're building a new feature to quickly ignore specific error messages.
+At first we will ship actions for eslint, stylelint, flake8, mypy and phpcs.
+
+Want to see actions for your favourite linter?
+Please open an issue so we can investigate it.
+
+Eventually this will become an API so that plugins can add their own.
```

Comment:
Well, yeah, LOL, that's too short.  We should say how you access the feature as an end-user. How a possible key binding should look like.  (T.i. how the main command is named.)  

Actually the API is very good to begin with, we just don't want to make any deprecation commitments at this early stage.  So we're tightly coupled because of that, and because at least in the beginning we need unit tests.  It is virtually impossible to write non trivial regexes without unit tests.  

On the other side, users can just import the decorator `@ignore_rules_inline` and use it even in User plugins for fun and testing.  The tricky part is what you write in such fixers, without the helper functions `extend_existing_comment` or `add_at_eol` it is challenging for non SublimeText freaks.  Maybe we need a lot of such helper functions, maybe not.  

So users can play with it, and just open PR's because maybe it is rather easy (like what you did with `phpcs`, anyone can write this). TL;DR **I don't want to investigate** it, which basically means I would have to write them anyway. Ideally we review good written PR's, *with* unit tests, and click merge. *With* tests so I don't have to test manually, we just merge.  The tests prove how the output will look. I don't want to review non-readable regexes, I see in the tests if they work. (And I would take that for the actual linters as well: copy paste the actual output of some linter, and prove that it gets transformed into our `LintError` correctly.)

Reply:
> Well, yeah, LOL, that's too short.

Actually, I don’t think so. People tend to ignore these messages, don’t even like to get them usually, and they cannot access the info after they close the message. It should just be a heads-up, with all the actual info in our readme, in the documentation. And how to use it should just be evident by using it. Nobody reads manuals anyway, and the update notification should definitely not be a manual.

I do get what you’re saying about PR’s though. I kinda doubt anyone will write tests though. Adding a quick action is trivial, writing tests isn’t, and isn’t something everyone knows how to do. But we’ll see what we’ll get. I’ll change the info to reflect that we want PR’s, not issues. Hopefully we can move the responsibility for these things to plug-in authors ASAP, so we don’t even have to be involved.